### PR TITLE
Refactor composer tutorial

### PR DESCRIPTION
**Pull request by koehnlein**
* Fixed images
* Moved "Requirements" to own page
* Corrected some typos

### CODE REVIEW

**Comment by DanielSiepmann** [comment]
Relates to TYPO3-Documentation/T3DocTeam#28

**Comment by DanielSiepmann** [comment]
@koehnlein can you grant us write access to your fork? This way we can update the pull request.

There is one small syntax issue regarding rst:
```diff
diff --git a/Documentation/MigrateToComposer/CoWorking.rst b/Documentation/MigrateToComposer/CoWorking.rst                                                                            
index c9c7c4d..33054fc 100644                                                                                                                                                         
--- a/Documentation/MigrateToComposer/CoWorking.rst                                                                                                                                   
+++ b/Documentation/MigrateToComposer/CoWorking.rst                                                                                                                                   
@@ -5,7 +5,7 @@ Co-working                                                                                                                                                            
 ==========

 Add to version control system
-----------------------------                                                                                                                                                         
+-----------------------------                                                                                                                                                        

 If you use a version control system like Git, it's important to add
 both files, the :file:`composer.json` and :file:`composer.lock` (which
```

Once fixed, I would merge the changes.

File: Documentation/MigrateToComposer/CoWorking.rst [hunk 1]
```diff
@@ -4,14 +4,15 @@
 Co-working
 ==========
 
-Add to version conrol system
+Add to version control system
 ----------------------------
```
Comment (DanielSiepmann) [inline Documentation/MigrateToComposer/CoWorking.rst]: The headline syntax is one to short, with the longer headline.

**Comment by koehnlein** [comment]
Wow. RST really has a strict syntax.

I fixed it. Would be great, if you could publish it soon. 👍

**Pull request merged**